src/Util.php

```
@@ -38,13 +38,13 @@ public static function urldecodeRfc3986($string)
      * parameters like this
      * array('a' => array('b','c'), 'd' => 'e')
      *
-     * @param mixed $input
+     * @param string $input
```

:+1: on this change, I think it was `mixed` before because `parse_url` technically can return `false` but that should probably raise an exception instead. I'll add a task to do that in the future.